packages/pratt-parser/pratt-parser.0.1.0-0/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+name: "pratt-parser"
+version: "0.1.0-0"
+maintainer: "Rizo Isrof <rizo@odis.io>"
+authors: "Rizo Isrof <rizo@odis.io>"
+homepage: "https://github.com/fold-lang/pratt-parser"
+bug-reports: "https://github.com/fold-lang/pratt-parser/issues"
+dev-repo: "https://github.com/fold-lang/pratt-parser"
+license: "BSD-3-Clause"
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "pratt-parser"]
```

Comment:
This is not enough because you install 2 executables but do not remove them.  Please use [oasis2opam](https://github.com/ocaml/oasis2opam) to generate correct OPAM files.